creating_packages/package_repo.rst

```diff
@@ -51,7 +51,7 @@ Now let's have a look at *conanfile.py*:
 
         def build(self):
             cmake = CMake(self)
-            cmake.configure(source_folder="src")
+            cmake.configure(build_script_folder="src")
```

Comment:
this belongs to the ``from conan.tools.cmake import CMake`` helper, not the one documented in this section ``from conans import CMake``